sdk/core/perf-tests/core-rest-pipeline/package.json

```
@@ -8,14 +8,21 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@azure/core-http": "^2.2.4",
     "@azure/core-rest-pipeline": "^1.1.0",
     "@azure/core-auth": "^1.3.0",
     "@azure/test-utils-perf": "^1.0.0",
-    "dotenv": "^8.2.0"
+    "node-fetch": "3.2.3",
```

```suggestion
    "node-fetch": "^3.2.3",
```

I would float `node-fetch` like we do for most other packages.